core/designsystem/src/main/java/com/mifos/core/designsystem/component/MifosEditTextField.kt

```diff
@@ -98,6 +102,61 @@ fun MifosOutlinedTextField(
     )
 }
 
+@Composable
```

Comment:
Why have you created this? It already exists in MifosEditTextField.

Reply:
@Aditya-gupta99  I overloaded that function so that we can use `string` directly instead of using `TextFieldValue.saver`. 
Should I still remove it?

Reply:
@itsPronay its already there in MifosTextField at line 103